web/src/p2k16/web/static/about.html

```diff
@@ -1,20 +1,27 @@
 <p2k16-header></p2k16-header>
 
-<div class="container">
-  <h1>About P2k16</h1>
+<div class="container col-xs-10 col-xs-offset-1 col-sm-6 col-sm-offset-3">
+  <h2 class="text-center">About P2K16</h2>
 
-  <p>
-    P2k16 is <a href="https://bitraf.no">Bitraf's</a> self-serve system. The code is available on
+  <br>
+
+  <p class="text-center">
+    P2k16 is <a href="https://bitraf.no">Bitraf's</a> self-serve system.<br>The code is available on
     <a href="https://github.com/bitraf/p2k16">Github</a>.
   </p>
 
-  <section ng-show="ctrl.gitRevision || true">
+  <!-- ================================================= -->
+  <!-- The following block is kept in case of future use -->
+
+  <!-- <section ng-show="ctrl.gitRevision || true">
```

Comment:
Comments like this are not very useful, stuff that is commented out is implicitly there for future use. 

Of course, in this case it should perhaps just be removed. It will always be available from git anyway.